genequery-rest/src/main/kotlin/gq/rest/services/OverlapService.kt

```diff
@@ -0,0 +1,52 @@
+package gq.rest.services
+
+import gq.core.data.GQModule
+import gq.core.data.Species
+import gq.core.data.intersectWithSorted
+import gq.core.genes.GeneFormat
+import gq.rest.GQDataRepository
+import gq.rest.exceptions.BadRequestException
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+data class OverlapResponse(val identifiedGeneFormat: String,
```

Comment:
We don't need information about neither `identifiedGeneFormat` nor `geneConversionMap` in this request. Can you please remove it? It's fine to leave just `overlapGenes`.